site/app/templates/course/CourseMaterials.twig

```diff
@@ -177,5 +177,5 @@
 
 
 {% include('admin/users/UserForm.twig') with {'action': 'update_student'} %}
-{% include('course/UploadCourseMaterialsForm.twig') %}
+{% include('course/UploadCourseMaterialsForm.twig') with {'max_size_string' : max_size_string} %}
```

Comment:
Included templates have access to the variables of the active context. `with` is for additional variables. `with {'max_size_string' : max_size_string}` is safe to be removed.

ref: https://twig.symfony.com/doc/2.x/tags/include.html